frontend/src/components/Form/SimpleFormInput.tsx

```diff
@@ -36,6 +36,29 @@ export default function SimpleFormInput<T extends FieldValues>({
   disabled = false,
   value,
 }: SimpleFormInputProps<T>) {
+  const callOnKeyUp = function <E>(event: React.KeyboardEvent<E>) {
```

Comment:
I don't quite get what this is supposed to achieve. Can you please elaborate. I think there might be a simpler solution.

Reply:
@absurd-turtle thx for the review! :heart: 

@badnames please also use code comments to describe something like this. But of course best if to have simple self-describing solutions (e.g. via functions and variables which have good names).

Reply:
These functions are supposed to translate keyboard events into string or number values before calling the onChange callback.
I need this additional functionality to update the map on each user input.

If there is an easier method of achieving these goals, feel free to tell me how exactly you would change this code.

Reply:
I think you can just remove the function entirely as you just pass the value to onChange either way but maybe there is something I am missing here.
I think it would be best to have a little chat about it :)